### PR TITLE
wayne_election_commission

### PR DESCRIPTION
**Pull request by novellac**
In progress - still working on the documents

### CODE REVIEW

**Comment by novellac** [comment]
References #500

**Comment by novellac** [comment]
Ready for review and comments

File: city_scrapers/spiders/wayne_election_commission.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        year_regex = re.compile(r'\d{4}')
+        year_str = year_regex.findall(year_xpath)[0]
+        month_day_str = item.xpath('td[1]//text()').extract_first()
+        try:
+            meeting_date = parse(month_day_str + ", " + year_str)
+            return {'date': meeting_date.date(), 'time': None, 'note': note}
+        except ValueError:
+            return {'date': None, 'time': None, 'note': note}
+
+    def _parse_sources(self, item):
```
Comment (mkrump) [inline city_scrapers/spiders/wayne_election_commission.py]: Doesn't look like using `_parse_sources` so can delete.

Comment (novellac) [reply city_scrapers/spiders/wayne_election_commission.py]: Oh yeah thanks. Done.

**Comment by novellac** [comment]
Ready for review, I think this one is good to go.

**Pull request merged**